src/styles/globals.css

```diff
@@ -90,6 +81,10 @@ body {
   .accent-gradient-4 {
     background: linear-gradient(270deg, #7FF5C5 0%, #1530CC 100%);
   }
+
+  .gradient-button {
+    background: linear-gradient(0deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), linear-gradient(72.57deg, #EC008C 17.15%, #5B10FF 89.41%, #0468D6 102.56%)
```

Comment:
can you check this gradient again

Reply:
checked against DSM its : `background: linear-gradient(0deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), linear-gradient(72.57deg, #EC008C 17.15%, #5B10FF 89.41%, #0468D6 102.56%);`

though on handover its accent-gradient-1 : `background: linear-gradient(72.57deg, #EC008C 17.15%, #5B10FF 89.41%, #0468D6 102.56%);`

should we follow DSM as source of truth

Reply:
updated to use accent-gradient-1 on default state and gradient-button-hover on hover